tasks/get_objects_for_app.yml

```diff
@@ -2,13 +2,31 @@
 
 - name: Set templates list for this app
   set_fact:
-    templates: "{{ app | json_query('services[*].templates[]') | list }}"
+    raw_templates: "{{ app | json_query('services[*].templates[]') | list }}"
   tags: deploy
 
+# Remove empty rendered templates (e.g. inactivated for an env_type)
+- name: Ignore empty rendered templates
```

Comment:
:crying_cat_face:

Reply:
Yes, we should work on this...

Reply:
I've declare an issue for this pain point, see #128